pkg/controller/containerrecreaterequest/crr_controller.go

```diff
@@ -60,6 +61,10 @@ var (
 	controllerKind       = appsv1alpha1.SchemeGroupVersion.WithKind("ContainerRecreateRequest")
 )
 
+func newPodReadinessControl(adp podadapter.Adapter) utilpodreadiness.Interface {
+	return utilpodreadiness.NewForAdapter(adp)
```

Comment:
Why re-define the func newPodReadinessControl, other than immediate call utilpodreadiness.NewForAdapter(adp)?

Reply:
done